pynamodb/models.py

```diff
@@ -915,6 +927,8 @@ def _get_save_args(self, condition: Optional[Condition] = None, *, add_version_c
             condition &= version_condition
         kwargs['attributes'] = attribute_values
         kwargs['condition'] = condition
+        if return_values_on_condition_failure and return_values_on_condition_failure is not None:
```

Comment:
This could be simplified to:

```suggestion
        if return_values_on_condition_failure:
```